### PR TITLE
[NCL-6328] Review regexps in rh-license-exceptions.json

### PR DESCRIPTION
**Pull request by thescouser89**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/bacon/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

File: pig/src/main/resources/rh-license-exceptions.json [hunk 1]
```diff
@@ -794,7 +794,7 @@
   {
     "groupId": "io.agroal",
     "artifactId": "agroal-api",
-    "version-regexp": "1.8*",
+    "version-regexp": "1\\.8*",
```
Comment (michalszynkiewicz) [inline pig/src/main/resources/rh-license-exceptions.json:797]: this looks a bit suspicious. Shouldn't it be sth like `1\\.8.*` ? (just a note, this change doesn't actually change the suspicious part)

**Review by michalszynkiewicz — APPROVED** [review]

File: pig/src/main/resources/rh-license-exceptions.json (hunk 1 above)
Comment (thescouser89) [reply pig/src/main/resources/rh-license-exceptions.json:797]: I'll have a look. Thanks for pointing this out!

Comment (thescouser89) [reply pig/src/main/resources/rh-license-exceptions.json:797]: agroal actually has version `1.8`, `1.9` etc.

`1\\.8.*` makes sense in that optic. Will do the change!

Comment (pkocandr) [reply pig/src/main/resources/rh-license-exceptions.json]: I would probably go for `1\\.8(\\..*)?` to not include e.g. 1.80 in a few years. WDYT?

**Pull request merged**